hooks/post_gen_project.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+GLITTER="{{ cookiecutter.glitter }}"
+
+if [ "$GLITTER" == "y" ]; then
+    # Glitter requested - use the glitter base HTML file with extra tags
+    mv templates/base_glitter.html templates/base.html
```

Comment:
Possibly. Need to figure out the out of the box glitter experience at some point.